app/views/content/returning-to-teaching.md

```diff
@@ -130,25 +138,26 @@ $q-helen-returner$
 
 [Read Helen's top tips for returning teachers](/my-story-into-teaching/returners/top-tips-for-returning-teachers).
 
-## National Tutoring Programme
+## If you've been out of teaching for a while
 
-The National Tutoring Programme supports schools to close the attainment gap in light of the coronavirus pandemic and provides different options to return to a school environment without taking on a full-time or part-time classroom role.
+If you’re returning to teaching, you might want to re-familiarise yourself with life in a classroom. Here are lots of ways that you can do it.
 
-If you’re unsure about taking on a teacher role at this time, [consider applying to tutor or mentor through the National Tutoring Programme](https://nationaltutoring.org.uk/) and help provide support to disadvantaged pupils.
+#### Reading teachers' stories and tips
 
-## If you've been out of teaching for a while
+* browse [inspirational stories](https://getintoteaching.education.gov.uk/blog/tag/returning-to-teaching) from Stephen, Farooq and Maria who returned to the classroom successfully after a break
+* read up on [teaching interview tips](https://www.teachertoolkit.co.uk/2019/04/04/job-interviews) and get [pointers on remote interviews](https://www.tes.com/news/coronavirus-10-tips-acing-remote-job-interview)
 
-There’s lots you can do to re-familiarise yourself with teaching. For example, you can:  
+#### Catching up on changes
 
-* attend an [online return to teaching event](/event-categories/online-q-as)
-* read [blog posts about returning to teaching](/blog/tag/returning-to-teaching)
-* read [Jill's journey](/my-story-into-teaching/returners/getting-back-into-the-classroom) to become a teacher again
 * review the latest [Teachers’ Standards](https://www.gov.uk/government/publications/teachers-standards)
 * catch up on changes to the [national curriculum and qualifications](https://www.gov.uk/topic/schools-colleges-childrens-services/curriculum-qualifications)
 * [read through current behaviour management guidelines](https://www.gov.uk/government/publications/behaviour-and-discipline-in-schools)
+* learn about how DfE have improved the [National Professional Qualifications (NPQs)](https://www.gov.uk/government/publications/national-professional-qualifications-npqs-reforms/national-professional-qualifications-npqs-reforms) and if you may be eligible for a fully funded scholarship 
+
+#### Things to do
```

Comment:
Is there a better way of saying this? All of the sections are things to do?

(But this is a minor comment which I don't feel strongly about - feel free to ignore!)

Reply:
originally, all of these links were bunched together in one section which was quite heavy, so I've tried a few different ways of splitting them up into three different sections. I've tried to think of a few different ways to section them, but like you said, there is quite a bit of crossover between them all, so happy to take on any suggestions for how to better group them?

Reply:
Breaking it up into sections is SO much better - definitely a good idea

Not sure I have a better suggestion though - Other things to do? Other things/actions to consider?

Actually I wonder if we need to say **consider** (or similar) rather than do - I wouldn't want to imply that they **have** to join a union or the Chartered College, for example